governance/permissionless-proposals.md

```diff
@@ -0,0 +1,80 @@
+- **Submission window check:** Ensure the proposal is submitted within a defined short window relative to the current voting cycle. (Type proposal check, this is not valid of all types of proposals)
+- **Type-aware validation:** Confirm that the proposal meets any additional criteria based on its type (e.g. higher thresholds for larger OP grants) as defined in the `ProposalTypesConfigurator`.
+- **Distribution rate limit**: Ensures the proposed amount does not go over the distribution rate limit.
+- **Forward valid proposals:** Once all checks pass, forward the call to the existing `OptimismGovernor` without reimplementing its core functions. To enable this, the `Top100DelegatesProposalValidator` must be granted proposal rights, which are currently restricted to the manager role.
+
+## The Workflow
+
+The general workflow would be:
+
+1. **Draft Proposal:** A delegate starts by creating a Draft proposal and posting it on the optimism governance forum. At this stage, the proposal is open to the entire community for feedback. 
+2. **Proposal Finalization**: After the proposal has been drafted, delegates provide feedback, ask clarifying questions, and propose changes. Once the delegate incorporates this input and the proposal is refined, an approved proposer will be able to call `propose` on the `Top100DelegatesProposalValidator` contract which will return a `proposalId` , aimed to be added into the forum proposal and mark it as “FINAL”.
+3. **Move to vote via Top100DelegatesProposalValidator:** With the final proposal ready, the top 100 delegates can start signing-off on the on-chain proposal using the `proposalId`. When the proposal is ready to move to voting anyone should be able to call `moveToVote` , and if the proposal meets the necessary rules, such as having the required top 100 delegates sign-offs, being submitted within the correct time window, not going over the distribution rate limit, and adhering to any proposal type-specific threshold it will be moved to the Governor for voting.
+4. **Governor:** If the `Top100DelegatesProposalValidator`confirms that the proposal is valid, it forwards the proposal to the OP Governor contract. Then, the voting time begins as usual.
+
+### Off-chain flow
+```mermaid
+  flowchart LR
+    Delegate([Delegate])
+    ForumDraft[OPGov Forum]
+    Community([Community])
+    ForumFinal[OPGov Forum]
+
+    Delegate -->|submit DRAFT proposal| ForumDraft
+    Community -->|provide feedback| ForumDraft
+    ForumDraft -->|submit FINAL proposal with proposalId| ForumFinal
+```
+
+### On-chain flow
+```mermaid
+flowchart LR
+    Proposer([Approved Proposer])
+    Validator[ProposalValidator]
+    TopDelegates([Top Delegates])
+    Executor([Executioner])
+    FinalValidator[ProposalValidator]
+    OPGov[OPGov]
+
+    Proposer -->|propose| Validator
```

Comment:
It seems like there is a tradeoff here between preventing a footgun where we validate certain things (e.g. OP amount) up front in this function call. However this does split validation logic between `propose` and `moveToVote`.  Maybe that's fine, feels worth discussing briefly

Reply:
Another question is whether this two-step flow should exist for proposal types which do not require the two steps (e.g. Maintenance Upgrade Proposal can be validated/submitted immediately)